src/Components/Dropdowns/Dropdown.tsx

```diff
@@ -22,6 +22,21 @@ export interface DropdownProps extends StandardFunctionProps {
   menu: (onCollapse?: () => void) => JSX.Element
   /** Renders the menu element above the button instead of below */
   dropUp?: boolean
+  /** Optional method that is triggered when the user clicks outside of/away from the dropdown */
+  onClickAway?: () => void
+  /**
+   * optional way for the owner of this dropdown to set the menu state:
+   * 'open':  the menu will be open
+   * 'closed': the menu will be closed
+   * any other string will be ignored.
+   *
+   * and the menu state (open/closed) only changes when the string changes.
+   * so the owner is responsible for resetting the string
+   * (if the string is set to 'open', and then the user closes it, and the code sets it to open again,
+   * unless the code sets it to something else in between (like 'ignore' or 'close'),
+   * then nothing will happen- the menu will not open)
+   * */
+  menuOpen?: string
```

Comment:
I expected `menuOpen` to be a boolean. I think `menuStatus` might be more descriptive. I think we should clamp the values with a TypeScript [string enum](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums) since it's a little bit tighter way of doing what you're going for.

something like

```ts
enum MenuStatus {
  Open = 'open',
  Closed = 'closed'
}

menuStatus?: MenuStatus
```

Reply:
then we would need a third one, for 'ignore'.  because it's not just the owner setting the open/close status, the user is controlling it too.

Reply:
...or could use null instead...  'setMenuStatus'  is a better name